lib/core/card-view/components/card-view-textitem/card-view-textitem.component.scss

```
@@ -129,11 +129,25 @@
             margin-bottom: -8px;
         }
 
-        &-textitem-ellipsis {
-            overflow: hidden;
-            text-overflow: ellipsis;
+        &-textitem-scroll {
+            overflow-x: scroll;
```

```suggestion
            overflow-x: auto;
```
please don't prefer `scroll`. it provides a scroll bar even content is not scrollable.  

https://stackoverflow.com/questions/35484742/hide-useless-scrollbars-that-show-up-on-windows-only